django_ibmi/base.py

```diff
@@ -18,7 +18,7 @@
 
 """
 DB2 database backend for Django.
-Requires: ibm_db_dbi (http://pypi.python.org/pypi/ibm_db) for python
+Requires: pyodbc for python
```

Comment:
```suggestion
Requires: pyodbc
```

For pyodbc, Python is implied.